recipes/scip/all/conanfile.py

```diff
@@ -76,11 +80,10 @@ def config_options(self):
 
     def requirements(self):
         if self.options.with_gmp:
-            self.requires("gmp/6.2.1")
+            self.requires("gmp/6.3.0")
         if self.options.with_sym == "bliss":
             self.requires("bliss/0.77")
-        self.requires("soplex/6.0.3")
-        self.requires("zlib/1.2.13")
```

Comment:
@hedtke Why did you remove zlib? I see it listed in scip files: https://github.com/scipopt/scip/blob/v804/src/xml/xmldef.h#L52

Reply:
it is automatically included by soplex

Reply:
@hedtke please, revert that change. Each package should manage their own dependencies, getting a hide over other dependencies to include a package will create a problem related to the package ID.

Reply:
Re-added and also using ranges instead of a specific version